weblate/accounts/notifications.py

```diff
@@ -446,8 +447,9 @@ class RepositoryNotification(Notification):
         Change.ACTION_REBASE,
         Change.ACTION_MERGE,
     )
-    # Translators: Notification name
-    verbose = gettext_lazy("Repository operation")
+    verbose = pgettext_lazy(
+        "Notification name", "Successful operation of the repository"
+    )
```

Comment:
This is any operation on the repository, so perhaps:

```suggestion
    verbose = pgettext_lazy(
        "Notification name", "Operation of the repository"
    )
```